osu.Framework.Tests/Visual/Platform/TestSceneDisplayMode.cs

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Drawing;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Platform;
+using osuTK;
+
+namespace osu.Framework.Tests.Visual.Platform
+{
+    [System.ComponentModel.Description("For complete validation, this test should run be with different WindowModes at startup, and with different resolutions in Fullscreen")]
```

Comment:
Can be vastly shortened:
```suggestion
    [System.ComponentModel.Description("For complete validation, run this with different window modes and resolutions")]
```